Source/WebCore/platform/mediastream/RealtimeMediaSourceCenter.cpp

```diff
@@ -205,7 +208,7 @@ void RealtimeMediaSourceCenter::getDisplayMediaDevices(const MediaStreamRequest&
         if (!device.enabled())
             return;
 
-        auto sourceOrError = displayCaptureFactory().createDisplayCaptureSource(device, String { hashSalt }, &request.videoConstraints, request.pageIdentifier);
+        auto sourceOrError = displayCaptureFactory().createDisplayCaptureSource(device, MediaDeviceHashSalts { hashSalts }, &request.videoConstraints, request.pageIdentifier);
```

Comment:
Ditto here and elsewhere.

Reply:
These factory functions each return a vector of capture devices and the salts are needed to construct each one.